sycl/test/fpga_tests/fpga_pipes.cpp

```diff
@@ -3,7 +3,7 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
-// UNSUPPORTED: cuda
```

Comment:
Does it pass on CUDA device?

Reply:
Yes because the test (as quite a few others) [has an extension check early on and passes if the extension is not found](https://github.com/intel/llvm/blob/sycl/sycl/test/fpga_tests/fpga_pipes.cpp#L299).

Reply:
Would you prefer to mark all the fpga tests as not supported by CUDA? I am not entirely sure if the test semantics only work for accelerators anyway, in which case I could also quickly add `REQUIRED: accelerator` to all of them.

Reply:
I'm fine with this change actually.
We probably should have some common guidance for test behavior in unsupported environment and align all the tests in a separate PR.
+@MrSidims